File: src/classes/api/req_res/login-response.ts
// ˅
'use strict';

import { GkillAPIResponse } from '../gkill-api-response';

// ˄

export class LoginResponse extends GkillAPIResponse {
    // ˅
    
    // ˄

    session_id: string;

    constructor() {
        // ˅
        super()
        this.session_id = "";
        
        // ˄
    }

    // ˅
    
    // ˄
}

// ˅

// ˄

File: src/app/gkill/api/req_res/login_response.go
// ˅
package req_res

import "github.com/mt3hr/gkill/src/app/gkill/api/message"

// ˄

type LoginResponse struct {
	// ˅

	// ˄

	Messages []*message.GkillMessage `json:"messages"`

	Errors []*message.GkillError `json:"errors"`

	SessionID string `json:"session_id"`

	// ˅

	// ˄
}

// ˅

// ˄
